docs/install/managing-multiple-projects.md

```diff
@@ -1,3 +1,103 @@
+or create additional identities in the form of Google service accounts for different Source custom objects,
+to achieve more fine-grained configurations)
+
+In general Knative-GCP installation guide, each Knative-GCP component consumes/creates Google Cloud resources within the scope of the project
+(the project where the kubernetes cluster contained your Knative-GCP components resides). For example, if your kubernetes cluster is in Project A, and
+you apply StorageSource in this kubernetes cluster, it will create a Google Cloud Storage Bucket, a Google Cloud PubSub Topic, and a Google Cloud PubSub Subscription,
+all belong to Project A.
+
+This topic explains how to set-up Knative GCP configuration to create Sources which consume/create Google Cloud resources from other projects.
+
+Here we use PubSubSource as an example.
+
+The following names are representing:
+* `project-a`: Project (project where all Knative-GCP system resides)
+
+* `events-controller-gsa@project-a.iam.gserviceaccount.com`: Control-plane GSA (which is used for the control plane)
+
+* `project-b`: Project (project where your topic and subscription resides)
+
+* `your-gsa@project-b.iam.gserviceaccount.com`: Source Data-Plane GSA (which is used in your namespace where your PubSubSource resides)
+
+In order to make your PubSubSource in Project A' cluster to consume Google PubSub resource in Project B:
+1. Control Plane GSA `events-controller-gsa@project-a.iam.gserviceaccount.com` needs `pubsub.editor` role for `project-b`
+    ```
+    gcloud projects add-iam-policy-binding project-b
+    --member=serviceAccount:events-controller-gsa@project-a.iam.gserviceaccount.com
+    --role roles/pubsub.editor
+    ```
+    ***Note:*** To create different Google Cloud Resource, the Control Plane needs different permissions.
+    Refer to [Manually Configure Authentication Mechanism for the Control Plane](./authentication-mechanisms-gcp.md/#authentication-mechanism-for-the-control-plane)
+    for specific roles(a set of permissions) needed to create each Source.
+
+1. Source Data-plane GSA `your-gsa@project-b.iam.gserviceaccount.com` needs `pubsub.editor` role for `project-b`
+
+    ```
+    gcloud projects add-iam-policy-binding project-b
+    --member=serviceAccount:your-gsa@project-b.iam.gserviceaccount.com
+    --role roles/pubsub.editor
+    ```
+   ***Note:*** Source Data Plane only pulls subscription from Cloud PubSub Subscription, thus,
+   no matter which Cloud Source you are creating, this GSA only needs `roles/pubsub.editor` role.
+   Refer to [Installing a Service Account for the Data Plane](../install/dataplane-service-account.md)
+   for specific roles(a set of permissions) needed if you also want metrics and tracing for your Source.
+
+1. Configure credentials in the kubernetes cluster. You can either use:
+    1. Workload Identity
+        1. Create a Kubernetes Service Account `ksa-name` in the same namespace where the PubSubSource will reside.
```

Comment:
```suggestion
        1. Create a Kubernetes Service Account `ksa-name` in the same namespace where the PubSubSource will reside.
```

If you use the shell variable idea earlier, make this a shell variable. 

In any case, tell users that they can choose any KSA name they want, `ksa-name` is just an example value.